src/model/apps.js

```diff
@@ -30,4 +32,4 @@ const AppSchema = new Schema({
 })
 
 export const AppModelAPI = connectionAPI.model('App', AppSchema)
-export const AppModel = connectionDefault.model('App', AppSchema)
+export const AppModel = connectionDefault.model('App', AppSchema)
```

Comment:
empty line at the end. Actually you need to run this:  `npm run lint:fix`
It will prettier and lint the code